ci/ctlplane_watcher_patch.yaml

```diff
@@ -1,9 +1,10 @@
 spec:
   watcher:
     enabled: true
-    decisionengineServiceTemplate:
-      customServiceConfig: |
-        [watcher_cluster_data_model_collectors.compute]
-        period = 60
-        [watcher_cluster_data_model_collectors.storage]
-        period = 60
+    template:
```

Comment:
this seam incorrect.

when we are deploying via the metadata operator we shoudl be patching the coolant cr not the watcher cr.

the watcher spec kind has a watcherspecCore type inline

https://github.com/openstack-k8s-operators/watcher-operator/blob/main/api/v1beta1/watcher_types.go#L30-L37

but that does not and should not have a tempelete filed

the template filed is part of the openstackCointolplane kind

where are you changin where this file is applied?

Reply:
i can see where this is updating the openstack version cr but not the openstackcontoleplan cr.

Reply:
looking at 
https://logserver.rdoproject.org/e34/rdoproject.org/e34c36252dbb48009259c6c3ceb68b42/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/crs/openstackcontrolplanes.core.openstack.org/controlplane.yaml

this deos seam to be workign but its not clear how.

Reply:
right os what was confusign is the change to aadd that patching is not part of this commit.

each commit shoudl be self contained and correct on its own but this commit is actully fixing incorrect code that laredy merged.

the other poitn of confugion at least to me was that you still have a depends on refecned in the commit message

Depends-On: https://github.com/openstack-k8s-operators/openstack-operator/pull/1535

that is not merged which is normally a blocker to merge the patch.